config/jobs/kubernetes/sig-scalability/sig-scalability-presets.yaml

```diff
@@ -81,6 +81,11 @@ presets:
     value: "false"
   - name: HOLLOW_PROXY_TEST_ARGS
     value: "--use-real-proxier=false"
+  - name: HEAPSTER_KUBELET_TEST_ARGS
+    value: "--register-with-taints=monitoring=:PreferNoSchedule"
```

Comment:
Can we switch to `NoSchedule` - with `PreferNoSchedule` we have exactly 0 guarantee for anything, and i don't want to end-up in a situation that 2 weeks later we will be debugging exactly the same problem.

Reply:
Done.

Reply:
Not done